ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Raw.scala

```diff
@@ -66,4 +66,11 @@ object Raw {
 
   type StateEntry = (StateKey, Envelope)
 
+  final case class Transaction(override val bytes: ByteString) extends Bytes
+
+  object Transaction extends Companion[Transaction]
+
+  final case class ContractInstance(override val bytes: ByteString) extends Bytes
+
+  object ContractInstance extends Companion[ContractInstance]
```

Comment:
(Out of scope here but it would be nice to use newtypes rather than wrappers at some point)

Reply:
As discussed offline, I would wait for Scala 3 opaque types.